src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/LanguageServiceTelemetryService.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.ProjectSystem.LanguageServices;
+
+namespace Microsoft.VisualStudio.Telemetry
+{
+    [Export(typeof(ILanguageServiceTelemetryService))]
+    internal class LanguageServiceTelemetryService : ILanguageServiceTelemetryService
+    {
+        private readonly ITelemetryService _telemetryService;
+
+        [ImportingConstructor]
+        public LanguageServiceTelemetryService(ITelemetryService telemetryService)
+        {
+            _telemetryService = telemetryService;
+        }
+
+        /// <summary>
+        /// Hashes personally identifiable information for telemetry consumption.
+        /// </summary>
+        /// <param name="value">Value to hashed.</param>
+        /// <returns>Hashed value.</returns>
+        public string HashValue(string value)
+        {
+            return _telemetryService.HashValue(value);
+        }
+
+        /// <summary>
+        /// Post a telemetry event when a design time build fails.
+        /// </summary>
+        /// <param name="projectId">A unique identifier for the project.</param>
```

Comment:
For interface implementation methods, if the docs are identical to those of the base I'd just remove them. There's less chance that they fall out of sync then.. The IDE will make documentation from the base visible here.